version/version.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import "fmt"
+
+const (
+	// VersionMajor is for an API incompatible changes
+	VersionMajor = 0
+	// VersionMinor is for functionality in a backwards-compatible manner
+	VersionMinor = 5
```

Comment:
Why `0.5.0-dev`?  As far as I can tell, there are still no runtime-tools releases, and [SemVer suggests starting with 0.1.0][1].  If we want SemVer, we should also document which API we're using to measure compatibility.  Is it the command line API?  The Go package APIs?  Both?  Without docs about which API is being validated, I'd rather have this package just declare an opaque string to match the Git tag, possibly with a `-dev` suffix between tagged commits.  For example:

```go
var Version = "0.1.0"
```

for the commit tagged 0.1.0,

```go
var Version = "0.1.0-dev"
```

for the commit after the 0.1.0 tag, and something like

```go
var Version = "unreleased"
```

for the current master (since we have no tags or releases at the moment).

[1]: http://semver.org/spec/v2.0.0.html#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase